src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_NUMBER = 101;
```

Comment:
Why you write 101 if we need 100?

Reply:
As I understood, according to the task, the maximum possible number on the ball should be 100, so in the .nextInt() method, I used the number 101 as the limit that is excluded. But maybe I just misunderstood this part of the task.